src/region/RegionList.tsx

```diff
@@ -20,8 +23,12 @@ export function RegionList({ activeId, annotations, className, onSelect, scale }
         return shapeA.height * shapeA.width > shapeB.height * shapeB.width ? -1 : 1;
     });
 
+    const handleOutsideClick = (): void => onSelect(null);
+
+    useOutsideClick(annotations, ref, handleOutsideClick);
```

Comment:
Seems like this could be simplified to: `useOutsideClick(ref, () => onSelect(null))`